modules/apps/27-interchain-accounts/controller/types/errors.go

```diff
@@ -6,5 +6,5 @@ import (
 
 // ICA Controller sentinel errors
 var (
-	ErrControllerSubModuleDisabled = sdkerrors.Register(SubModuleName, 2, "controller submodule is disabled")
+	ErrControllerSubModuleDisabled = sdkerrors.Register(SubModuleName, 1, "controller submodule is disabled")
```

Comment:
Any ideas why this was changed? Isn't `1` reserved? See: https://github.com/cosmos/cosmos-sdk/blob/main/errors/errors.go#L17

cc. @colin-axner

Reply:
yea that's a bug. 1 is reserved for sdk, it'll discard our error messages otherwise

Reply:
nice catch